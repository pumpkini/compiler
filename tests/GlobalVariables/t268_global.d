int a;

int foo(int a,  bool c) {
    if (c){
	  Print("ok");
      return a + 2;
    }
    else
      Print(a, " wacky.");
   return 18;
}

void main() {
	int b;

	a = 10;
	b = a/2;
	foo(a,  true);
	foo(b + 2, a <= b);
	foo(foo(3, true && false),  !true);
}