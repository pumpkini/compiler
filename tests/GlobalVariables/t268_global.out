ok
7 wacky.
3 wacky.
18 wacky.
